endorsement/storeapi.proto

```diff
@@ -44,6 +63,58 @@ message OpenArgs {
 message CloseArgs {
 }
 
+message SwComponentID {
+    AttestationFormat type = 1;
+    google.protobuf.Struct parts = 2;
+}
+
+message SwComponent {
+    SwComponentID id = 1;
+    google.protobuf.Struct meta = 2; 
```

Comment:
I thought we agreed on having meta in addition to the (id, value), similar to TrustAnchor.